src/main/java/Event.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+
+public class Event extends Task {
+    /**
+     * Sets the task description and it's task type to Events.
+     *
+     * @param description description of the task
+     */
+    public Event(String description) {
+        super(description + ")");
+        this.taskType = "E";
+    }
+
+    /**
+     * Returns the newly added task in the list and the number of remaining tasks to complete.
+     *
+     * @param tasks a list used to store the tasks
+     * @param userInput user input
+     * @param numberOfTasks number of tasks in the last
+     * @return newly added task in the list and the number of remaining tasks to complete
+     */
+    public static String printEvent(ArrayList<Task> tasks, String userInput, int numberOfTasks, int remainingTasks) {
+        String event = userInput.substring(5).replace("/at","(at:");
+        tasks.add(new Event(event));
+        return "    Ok I add this task to ur list liao:\n" + "    " + tasks.get(numberOfTasks - 1).getTaskType()
```

Comment:
A reminder to change "\n" to System.lineSeparator() which is more appropriate. If you find it hard to use consider making it a constant. Example: private static final String LS = System.lineSeparator(); and printing LS when line separator is needed.

Reply:
Thank you for the feedback! I have refactored all the "\n" into System.lineSeparator() as suggested.